src/main/java/github/weichware10/analyse/gui/analyse/Analyzer.java

```diff
@@ -168,30 +178,41 @@ public static void analyse() {
                 diaConfig.toString());
         Logger.info(output);
```

Comment:
kannst du die ganzen logs bitte noch rausnehmen?

Reply:
git the bug xD